### ISSUE TITLE
Widgets stay always on top

### ISSUE THREAD

**Issue by XayOn** [akshat46/perpetual-zen#10]
I'm wondering if this is the expected behaviour.

Widgets and "banners" always stay on-top and visible. That might make sense with the toggable widgets like top and spawtify, but not so much on the banners like the tag list and the wifi+time+battery+volume indicators.

I'm could do a PR on this making them on_top=False, but I think they might be useful when changing screens (I believe you mentioned something like that on reddit).

**Comment by akshat46** [comment]
Yes, its expected behavior. An improvement of this could be to keep the banners on top of all clients except the ones that are fullscreen. With maybe a keybind to bring the banners on top.

**Comment by XayOn** [comment]
I've added a few screenshots depicting the issue:
https://imgur.com/a/bUAArHZ

The first one fits perfectly on your use-case, making banners go away on fullscreen clients would solve it.
The second one is the default behaviour, wich looks really nice on screenshots.
The third one is the same as the second one... with two terminals, rendering a corner of one of those terminals unusable, and part of the other. That makes both terminals virtually unusable for anything serious.

I can come up with a few options to "fix" that issue, but I'm not really good on ux.

- Reducing banners' height and fixing all windows to fit in the remaining space.
- Toggle visibility button for both banners on demand with a key binding
- Toggle visibility on left banner only when we're changing workspaces and right banner on-demand with a key binding

**Comment by akshat46** [comment]
First option would probably defeat the purpose of the whole "banners" idea. Which was to allow work area to be using entire screen at all times with ability to toggle information visibility whenever needed. 
Second and third options sound good. Specially third! Displaying left banner with a behavior similar to layout popup's would be great.
